src/utils/ENUM/lotto.js

```diff
@@ -0,0 +1,21 @@
+export const LOTTO_RANK_RULES = {
+  FIRST: 6,
+  SECOND: 5,
+  THIRD: 5,
+  FOURTH: 4,
+  FIFTH: 3,
+};
+
+export const LOTTO_RANK_MONEY = {
+  FIRST: 2000000000,
```

Comment:
큰 숫자를 적을 때는 numeric seperator를 이용해보셔도 좋을거 같아요 ! (저는 네자리 넘어가면 항상 씁니다 ㅋㅋ)

```suggestion
  FIRST: 2_000_000_000,
```

Reply:
[d0f72ac](https://github.com/next-step/js-lotto/pull/298/commits/d0f72acb38fc3187f4001b44add45959c0591e41)
수정 하였습니다~